solanaj/src/main/java/org/p2p/solanaj/ws/SignatureNotification.kt

```diff
@@ -0,0 +1,7 @@
+package org.p2p.solanaj.ws
+
+class SignatureNotification(val error: Any?) {
+    fun hasError(): Boolean {
```

Comment:
can be converted to a field. the class is immutable, this field can be assigned easily